features/facebookImport/test/utils/data-importing.js

```diff
@@ -2,12 +2,18 @@ import MessagesImporter from "../../src/model/importers/messages-importer.js";
 import NameImporter from "../../src/model/importers/name-importer.js";
 import PersonalDataImporter from "../../src/model/importers/personal-data-importer.js";
 import FacebookAccount from "../../src/model/entities/facebook-account.js";
-import { runImporter, runImporters } from "@polypoly-eu/poly-import";
+import {
+    runImporter,
+    runImporters,
+    runOutdatedImporter,
```

Comment:
Outdated is clear enough, but a small docstring for this will help the IDE identify it as deprecated.

Reply:
It's just for testing and will be gone very soon. Can add it though